tests/smoke.py

```
@@ -0,0 +1,174 @@
+    except (TypeError, ValueError):
+        return None
+
+
+def has_content(value):
+    return isinstance(value, str) and bool(value.strip())
+
+
+def check_health():
+    try:
+        status, _, body = request("/health", auth=False)
+    except Exception as error:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)